--- a/capacitor.config.ts
+++ b/capacitor.config.ts
@@ -1,3 +1,5 @@
+/// <reference types='@cap-kit/device' />
+
 import type { CapacitorConfig } from '@capacitor/cli';
 import { KeyboardResize } from '@capacitor/keyboard';
 
@@ -25,6 +27,14 @@
     Keyboard: {
       resize: KeyboardResize.None,
     },
+    Device: {
+      /**
+       * Enables verbose native logging.
+       * Logs are printed in Logcat (Android) and Xcode console (iOS).
+       * Default: false
+       */
+      verboseLogging: false,
+    },
   },
   cordova: {},
 };
