--- a/src/app/plugins-list.ts
+++ b/src/app/plugins-list.ts
@@ -13,2 +13,14 @@
 export const PLUGINS: PluginConfig[] = [
+  {
+    id: 4,
+    name: 'Integrity',
+    npmName: '@cap-kit/integrity',
+    iconName: 'shield-half',
+    githubUrl:
+      'https://github.com/cap-kit/capacitor-plugins/tree/main/packages/integrity',
+    pathName: '/integrity',
+    description: 'Runtime integrity and environment signal detection.',
+    active: true,
+    isVisible: true,
+  }
 ];
